changelogs/fragments/skip_invalid_coll_name_when_listing.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - Skip invalid collection names when listing in ansible-doc instead of throwing exception. Issue#72257
```

```suggestion
  - Skip invalid collection names when listing in ansible-doc instead of throwing exception (https://github.com/ansible/ansible/issue/72257).
```
See https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to